data-raw/parasite_parameters.csv

```diff
@@ -0,0 +1,4 @@
+parasite,dd,dt,da,du,del,dl,dpl,mup,mum,du_max,du_min,ku,au50,rm,rvm,rb,rva,rc,rid,b0,b1,ib0,kb,b,phi0,phi1,ic0,kc,theta0,theta1,kv,fv0,av,gammav,iv0,fd0,ad,gammad,d0,d1,id0,kd,ub,uc,uv,ud,cd,gamma1,cu,ct,ca,a0,rho,sigma_squared,n_heterogeneity_groups,de,delay_gam,dem,pcm,pvm,init_ica,init_iva,init_icm,init_ivm,init_id,init_idm,init_ib,f,gammal
```

Comment:
If we go this route - would this csv be a good place to store the referecnes for the default parameters. You could image it set up in long form such as:

| parasite | parameter | default | reference |

Reply:
I had this thought too. I think we would need to be a little careful. I originally had the parameter data in long form, but because the values contain numerics and strings, they all revert to strings (which they don't do in wide form). Adding a reference to each column would have the same problem. There could be a way around this though...? The parameters also contain vectors and empty lists and numerics which I also couldn't work out how to include in a csv, so I gave up on trying to include them and settled with doing just the parasite parameters.